spark/src/main/scala/geotrellis/spark/io/hadoop/HadoopAttributeStore.scala

```diff
@@ -34,8 +34,12 @@ import scala.util.matching.Regex
 class HadoopAttributeStore(val rootPath: Path, val hadoopConfiguration: Configuration) extends BlobLayerAttributeStore {
   import HadoopAttributeStore._
 
-  val attributePath = new Path(rootPath, "_attributes")
-  val fs = attributePath.getFileSystem(hadoopConfiguration)
+  val (fs, attributePath) = {
+    val ap = new Path(rootPath, "_attributes")
+    val fs = ap.getFileSystem(hadoopConfiguration)
+    // Get the absolute path to attributes
+    (fs, fs.getFileStatus(ap).getPath)
```

Comment:
A straightforward solution would be to use try catch, and to create dirs on exception. 
Another solution would be to `getFileStatus` of a root path and to create only `_attributes` folder, but that would require manual catalog creation, and still would throw runtime exceptions.

```scala
val (fs, attributePath) = {
    val ap = new Path(rootPath, "_attributes")
    val fs = ap.getFileSystem(hadoopConfiguration)
    // Get the absolute path to attributes
    try {
      fs.getFileStatus(ap)
    } catch {
      case _: FileNotFoundException => fs.mkdirs(ap)
    }

    (fs, fs.getFileStatus(ap).getPath)
  }
```